mantle/kola/harness.go

```diff
@@ -103,7 +104,7 @@ var (
 	Tags            []string // tags to be ran
 
 	extTestNum  = 1 // Assigns a unique number to each non-exclusive external test
-	failedTests []*register.Test
+	failedTests protectedFailedTests
```

Comment:
It looks like we are unconditionally storing test results in this data structure and then evaluated at the end if they passed or failed by iterating over the list. Maybe we should adjust our naming to something like:

```
TestResults protectedTestsResults
```